utils/methods.go

```diff
@@ -15,20 +15,21 @@ import (
 	"os"
 	"path/filepath"
 	"regexp"
+
 	"strconv"
 	"strings"
 
-	"github.com/chromedp/cdproto/network"
 	"github.com/chromedp/cdproto/cdp"
+	"github.com/chromedp/cdproto/network"
 	"github.com/chromedp/chromedp"
 )
 
+var Headless = true
+
 // Initializes Chrome DevTools Protocol
 func InitChromeDp() (chromedpCtx context.Context, cancelFnc context.CancelFunc) {
 	log.Printf("Initializing chromedp...")
-	headlessEnv, present := os.LookupEnv("HEADLESS_MODE")
-	doHeadless, _ := strconv.ParseBool(headlessEnv)
```

Comment:
This is where `strconv` was used, the import for it needs to be removed to allow the build checks to pass :)

Reply:
Ah ok thanks! I'm going to make another pull request changing that.

Reply:
Or wait I think you can see the change.

Reply:
Looks good now, thanks!